### PR TITLE
[14.0][MIG] pos_picking_delayed

### PR DESCRIPTION
**Pull request by em230418**

### CODE REVIEW

File: pos_picking_delayed/models/pos_order.py [hunk 1]
```diff
@@ -20,23 +20,28 @@ def create_from_ui(self, orders, draft=False):
...
+            order_data["data"]["has_picking_delayed"] = (
+                session.config_id.picking_creation_delayed
+                and not session.update_stock_at_closing
+            )
         return super(PosOrder, self.with_context(create_from_ui=True)).create_from_ui(
             orders, draft=draft
         )
 
-    def create_picking(self):
+    def _create_order_picking(self):
```
Comment (legalsylvain) [inline pos_picking_delayed/models/pos_order.py:35]: Why this rename ?

Comment (em230418) [reply pos_picking_delayed/models/pos_order.py:35]: Follow up of https://github.com/odoo/odoo/commit/2f2bde5d9f647e4785ffabdc1b2e07a08722309c#diff-29cbaebb5b63b539ab173d9340b2aec87b9ad63cd322eec2347879c5412bd50b

Comment (legalsylvain) [reply pos_picking_delayed/models/pos_order.py:35]: hum... that makes sense ! sorry for the naive question.

**Comment by em230418** [comment]
Hello, @legalsylvain!
I suggest to also merge this after merging version of 16.0

**Pull request merged**